Tools/wasm/python.html

```diff
@@ -123,28 +124,44 @@
         this.xterm.open(container);
     }
 
-    handleReadComplete(lastChar) {
-        this.resolveInput(this.input + lastChar)
-        this.activeInput = false
-    }
-
     handleTermData = (data) => {
-        if (!this.activeInput) {
-            return
-        }
         const ord = data.charCodeAt(0);
-        let ofs;
+        data = data.replace(/\r\n|\r|\n/g, "\n")  // Normalize line endings
```

Comment:
What about multilines """ """ and ''' ''' if they contain \r\n they are not meant to be replaced as they are not source code but string data. as \r\n in source will not bother the tokenizer i'd suggest to just do nothing.

Reply:
My web browser seems to `\r` line endings when I paste text (even though the text I copied has `\n` endings). When just `\r` is pasted, xterm.js seems to get upset (all the text shows up on one line). I'll remove the `\r\n` normalizing but keep the `\r` normalizing.

Reply:
Ubuntu Linux 20.04